isis/src/osirisrex/tsts/mapcam/Makefile

```diff
@@ -18,18 +18,12 @@ commands:
 	ocams2isis $(TSTARGS)                                                                        \
 	           from=$(INPUT)/20190303T100344S990_map_iofL2pan_V001.fits                          \
 	           to=$(OUTPUT)/20190303T100344S990_map_iofL2pan_V001-spiceinit.cub                  \
-	           >& /dev/null;
+	           > /dev/null;
```

Comment:
I removed the ampersands to allow errors to print out for debugging purposes. Seems like leaving the ampersands is actually better as we can get a better idea as to why something failed in Jenkins logs? 

If anyone disagrees or there is another reason to re-route stderr to /dev/null, I can put them back in.